eng/ignore-links.txt

```diff
@@ -4,3 +4,4 @@ https://github.com/Azure/azure-digital-twins/blob/private-preview/Documentation/
 https://github.com/Azure/azure-sdk-for-net-pr/tree/feature/IoT-ADT/sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DTDL/DigitalTwins
 https://github.com/Azure/azure-sdk-for-net-pr/blob/feature/IoT-ADT/sdk/iot/Azure.Iot.DigitalTwins/samples/DigitalTwinServiceClientSample/DTDL/Relationships/HospitalRelationships.json
 https://www.nuget.org/packages/Azure.ResourceManager.DigitalTwins
+https://contoso.azureedge.net/urlsigning/test?expires=2145916800&amp;keyid:key1&amp;signature=iTsrLX9rVAIJkSahBA_j5o9Azf5-j331ohxDR1Gx2js=
```

Comment:
Why is this link being added to the ignore list?

Reply:
Am not sure what it does but it is not a broken link. It does link to something.

Reply:
@sima-zhu can you please investigate to see if there is something wrong with our link checker for this link?